.github/workflows/build-images.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: Build Docker images for all OpenSlides services
+on: push
+env:
+  IMAGE_VERSION: 3.4.0-dev
+jobs:
+  build:
+    name: Builds Docker images
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Build images
+      run: docker/build.sh -D ghcr.io/${{ github.repository_owner }} -t $IMAGE_VERSION all
```

Comment:
> Maybe "lastest" should be the default

This would be the docker-standard. For releases explicit tags should be set.

Reply:
Done

Reply:
I strongly recommend that we either use latest only for the OpenSlides 4+ version or that we rename all OpenSlides3 images so that the image name contains the "3" (openslides3-server, openslides3-autoupdate, ...).

Reply:
I discussed it with gernot some weeks ago. We decided to use `latest4` for OS4. If we mainly are on OS4, it will become the latest.

Reply:
Please use `4.0.0-dev` or `4.0.0-` combined with any other nice string to be compatible with semver. The tag `latest4`would be invalid and may cause problems with some tools we might use. Also it is not common ...